nf_core/modules/module_test.py

```diff
@@ -71,25 +85,66 @@ def run(self):
     def _check_inputs(self):
         """Do more complex checks about supplied flags."""
 
+        # Retrieving installed modules
+        if self.repo_type == "modules":
+            local = False
+            installed_modules = self.module_names["modules"]
+        else:
+            local = questionary.confirm("Is the module local?", style=nf_core.utils.nfcore_question_style).unsafe_ask()
+            if local:
+                installed_modules = self.module_names.get("local/modules")
+            else:
+                installed_modules = self.module_names.get("nf-core/modules")
```

Comment:
```suggestion
                installed_modules = self.module_names.get(self.modules_repo.fullname)
```
Otherwise we will just run the tests on the nf-core modules regardless of the `--git-remote` flag.